libvips/foreign/dzsave.c

```diff
@@ -2009,38 +2009,23 @@ vips_foreign_save_dz_build(VipsObject *object)
 		return -1;
 	}
 
-	/* Default to white background. vips_foreign_save_init() defaults to
-	 * black.
-	 */
-	if (!vips_object_argument_isset(object, "background")) {
-		VipsArrayDouble *background;
-
-		/* Using g_object_set() to set an input param in build will
-		 * change the hash and confuse caching, but we don't cache
-		 * savers, so it's fine.
-		 */
```

Comment:
Ah, I just noticed this comment. Perhaps all those FIXMEs in the savers can be removed?